ext/Geant4/g4jl_application.jl

```diff
@@ -66,31 +70,39 @@ function SSDGenerator(source::SolidStateDetectors.MonoenergeticSource;  kwargs..
     data = GeneratorData(;kwargs...)
 
     function _init(data::GeneratorData, ::Any)
-        gun = data.gun = move!(G4ParticleGun())
+        data.gun = move!(G4SingleParticleSource())
         particle = data.particle = FindParticle(source.particle_type)
-        data.position = G4ThreeVector(
+        SetParticleDefinition(data.gun, particle)
+
+        # Place the SingleParticleSource as point source
+            data.position = G4ThreeVector(
```

Comment:
Would be nice if this line is unindented by one level

Reply:
Done!